users/tests.py

```diff
@@ -67,21 +165,70 @@ def test_for_role(self):
 class OrganizationUserTestCase(BaseTestCase):
     def setUp(self):
         super().setUp()
-        # set up an organization
-        self.organization = Organization.objects.create(name="Org 1", shortcode="org-1")
-        # set up a user that's supposed to be in the organization
-        self.organization_user = User.objects.create(mobile="+919988776655")
+
+        # create another organization
+        self.organization_2 = Organization.objects.create(
+            name="Org 2", shortcode="org-2"
+        )
+
+        # seed some organization users
+        self.org_user_1 = OrganizationUser.objects.create(
+            organization=self.organization, user=self.user, role=self.org_view_role
+        )
+
+        self.org_user_2 = OrganizationUser.objects.create(
+            organization=self.organization_2, user=self.user, role=self.org_view_role
+        )
+
+    def test_superuser_can_list_all_organization_users(self):
+        # make the current user as superuser
+        self.user.is_superuser = True
+        self.user.save()
+
+        # get organization users
+        response = self.client.get(reverse("organization-users-list"))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.json()), 2)
+        self.assertEqual(response.json()[0]["user"], self.user.id)
+        self.assertEqual(response.json()[1]["user"], self.user.id)
+        self.assertEqual(response.json()[0]["organization"], self.organization.id)
+        self.assertEqual(response.json()[1]["organization"], self.organization_2.id)
+
+    def test_normal_user_cannot_list_organization_users(self):
+        # get organization users
+        response = self.client.get(reverse("organization-users-list"))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.json()), 0)
```

Comment:
This should actually not return 200 :P 
Not allowed or bad request or some error code should come, ideally